client/src/components/common/Footer/index.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import './style.css'
+
+
+const Footer= () => {
```

Comment:
Make sure that you're all running the same linter / formatter otherwise it'll make pull requests harder to read (if someone using prettier edits a file written by someone who doesn't, for example, the file will be full of changes even if they only change one line)